src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -254,7 +254,15 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
         GPathResult parsedJobs = new XmlSlurper().parseText(resultLines.last())
 
         parsedJobs.children().each { it ->
-            GenericJobInfo gj = new GenericJobInfo(it["Job_Name"] as String, null, it["Job_Id"] as String, null, it["depend"] ? (it["depend"] as  String).find("afterok.*")?.findAll(/(\d+).(\w+)/) { fullMatch, beforeDot, afterDot -> return beforeDot } : null)
+
+            BEJobID jobID
+            try{
+                jobID = new BEJobID(it["Job_Id"] as String)
+            }catch (Exception exp){
+                throw new BEException("Job ID '${it["Job_Id"]}' could not be transformed to BEJobID ")
+            }
+
+            GenericJobInfo gj = new GenericJobInfo(it["Job_Name"] as String, null, jobID, null, it["depend"] ? (it["depend"] as  String).find("afterok.*")?.findAll(/(\d+).(\w+)/) { fullMatch, beforeDot, afterDot -> return beforeDot } : null)
```

Comment:
You could pull the code for the dependencies in the previous line to make it more readable